pkgs/tools/networking/goodhosts/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub, testers }:
```

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, testers
}:
```
just a small formatting nit

---

done